### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by bohdanbuha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,4 +15,4 @@ AI supports next complexity levels: Random move, Win strategy
 
 Actions: New, Save, Load
 
-The proposal for data driven approach: configuration (saved in configuration file or DB): board size, distance, tool size, win statement, lose statement, AI parameters
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:18]: 1) wrong repo name
2) please revert all changes and create correct pull request
3) tag is wrong

**Review by andrii-s-shpitser-lpnu-ua — CHANGES_REQUESTED** [review]

**Comment by bohdanbuha** [comment]
Edited repo name and git tag

File: README.md (hunk 1 above)
Comment (bohdanbuha) [reply README.md:18]: Edited repo name and git tag

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Comment by andrii-s-shpitser-lpnu-ua** [comment]
Ihave missed that your branch are wrong - please - fix

**Pull request merged**